### PR TITLE
Implementing issue 7

### LINKED ISSUE #7: Clarify that no candidate needs to be sent or received before starting timer (Technical)
WHERE: Update to RFC 8445

WHAT:

Modify...

"; note that receipt of an ICE candidate is not required."

...so that it also indicates that no candidate needs to be SENT before the timer is started.

WHY: The text in unclear on whether a candidate needs to be sent or not before the timer is started. The text only says that a candidate does not need to be received.

### PR DESCRIPTION
**Pull request by cdh4u**
Fixes #7

### CODE REVIEW

File: draft-ietf-ice-pac.xml [hunk 1]
```diff
@@ -187,7 +187,8 @@
         <xref target="RFC8445" />, Section 5.3)
         and has received some indication that the remote side is
         ready to start connectivity checks, typically via receipt of the values
-        mentioned above; note that receipt of an ICE candidate is not required.
+        mentioned above. Note that the ICE agent will start the timer even if it
```
Comment (juberti) [inline draft-ietf-ice-pac.xml]:
```suggestion
        mentioned above. Note that the agent will start the timer even if it
```

File: draft-ietf-ice-pac.xml [hunk 2]
```diff
@@ -187,7 +187,8 @@
...
+        has not yet sent or received any ICE candidates.
```
Comment (juberti) [inline draft-ietf-ice-pac.xml]:
```suggestion
        has not sent or received any ICE candidates.
```

**Review by juberti — APPROVED** [review]
LG with two minor wording changes

**Comment by cdh4u** [comment]
> LG with two minor wording changes

I committed both changes.

**Pull request merged**